src/main/java/frc/robot/subsystems/scoring/ShooterIO.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems.scoring;
+
+import org.littletonrobotics.junction.AutoLog;
```

Comment:
Do we need to put the `@AutoLog` decorator on stuff in this file (and in the other IO files)?

Reply:
AutoLog puts the data into the logs and advantagescope

Reply:
AutoLog specifies that we are logging to AdvantageScope and the logs (6328 does it)

Reply:
somehow I missed that you used the decorator here lol